Resources/Prototypes/Entities/Objects/Weapons/Guns/Launchers/launchers.yml

```diff
@@ -7,8 +7,13 @@
   components:
   - type: Sprite
     netsync: false
-  - type: Item
+  - type: Clothing
     size: 60
```

Comment:
```suggestion
  - type: Item
    size: 60
  - type: Clothing
```

Reply:
see clothing/item refactor in #codebase-changes

Reply:
I thought this was the idea, it errored when I pulled master because they no longer had an item component since this PR changed them all to clothing :death: